daemon/mgr/spec.go

```diff
@@ -30,6 +30,7 @@ var setupFunc = []SetupFunc{
 	setupCap,
 	setupNoNewPrivileges,
 	setupOOMScoreAdj,
+	setupUlimits,
```

Comment:
I wish that we could merge https://github.com/alibaba/pouch/pull/1154 first. This refactoring is quite important I think. I suggest promoting that pr's priority. 

Otherwise conflict is there.